src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2768,6 +2768,12 @@ void ApplicationManagerImpl::UnregisterApplication(
     MessageHelper::SendStopAudioPathThru(*this);
   }
 
+#ifdef SDL_REMOTE_CONTROL
+  if (!is_resuming) {
```

Comment:
@AKalinich-Luxoft 
I don't think you need this check. If application is unregistered no matter of what reason plugin must be notified